src/setupTests.js

```diff
@@ -11,3 +11,13 @@ const localStorageMock = {
   clear: jest.fn()
 }
 global.localStorage = localStorageMock
+
+global.console = {
+  log: jest.fn(),
+  error: jest.fn(),
```

Comment:
I guess it's worthwhile keeping `warn`s in tests for now. We can disable those too if they prove too noisy